CHANGELOG.md

```diff
@@ -413,4 +414,4 @@ LineTouchData(
 * restricted to access private classes of the library
 
 
-## 0.0.1 - Released on (2019 June 4)
\ No newline at end of file
+## 0.0.1 - Released on (2019 June 4)
```

Comment:
Why this line has changed? Can you revert this change?

Reply:
It seems that a newline character has been added to the end of this line, hence why it seems to have been edited. I'm busy figuring out how to undo that part of the commit

Reply:
Ok I tried to remove the newline character and committed it, but it seems it was unsuccessful. Are you able to change it from your side?